aaf_bringup/scripts/aaf_start.sh

```diff
@@ -47,7 +47,7 @@ tmux select-window -t $SESSION:5
 tmux send-keys "DISPLAY=:0 roslaunch aaf_bringup aaf_navigation.launch map:=/opt/strands/map/aaf_deployment16-cropped.yaml topological_map:=aaf_deployment16"
 
 tmux select-window -t $SESSION:6
-tmux send-keys "DISPLAY=:0 roslaunch perception_people_launch people_tracker_robot.launch machine:=werner-left-cortex user:=strands"
+tmux send-keys "DISPLAY=:0 roslaunch perception_people_launch people_tracker_robot.launch machine:=werner-left-cortex user:=strands with_wheelchair_detector:=true wheelchair_machine:=werner-left-cortex wheelchair_user=strands"
```

Comment:
`wheelchair_user=strands` should be replaced by `wheelchair_user:=strands` second time this happens to me